src/app/create-edit/create-edit.component.html

```diff
@@ -95,6 +95,7 @@
                 </ion-item-group>
                 <div class="ion-text-center">
                     <ion-button
+                        type="submit"
```

Comment:
this causes a weird bug on iOS, see https://github.com/chris-nagy98/unigrade-app/issues/12, please revert this again.